### PR TITLE
fixing documentation for .stdev and .variance

### PR DESCRIPTION
**Pull request by jamescgibson**
Fix problem reported via the Google Group. The tests were correct; the functions were incorrectly documented.

### CODE REVIEW

**Comment by jamescgibson** [comment]
@trevnorris If I could impose on you to give a brief howto on merging things into a version branch / master / &c, that'd be much appreciated - don't want to mess it up. 

Thanks!

File: doc/md/vector.md [hunk 1]
```diff
@@ -708,7 +708,7 @@ less or equal than (if `kind == 'weak'`) score. Default is `'weak'`.
 
 **histogram( dataArray, [numBins] )**
```
Comment (trevnorris) [inline doc/md/vector.md]: Since you're here, mind changing this to:
```
**histogram( dataArray[, numBins] )**
```

Comment (trevnorris) [inline doc/md/vector.md]: actually, nm it. seems the file needs a pass of fixes anyway.

**Comment by trevnorris** [comment]
LGTM

Now, merge tips:

* Check fields in previous commits (e.g. 7fb9fe04d) for a couple of the required fields. Make sure to use the _full_ URL in `PR-URL:`, and add my info in `Reviewed-by:`.
* Make sure commit message and message body are free of typo's. Commit message max of 50 chars, and message body max line length of 72 chars.
* Since this is only a single commit can edit the commit message by doing `git commit --amend` and edit the commit message that way.

Let me know if you have any questions, or if you'd like me to take care of it.

**Comment by jamescgibson** [comment]
Great. I'll do it asap, I would like to get the hang of it so you don't need to do all the merging. Thanks!

Is there anything in particular that needs to be done to get the documentation onto the github pages, or does that update automatically? I've never used github pages.

**Comment by trevnorris** [comment]
@jamescgibson I just generate the documentation and update the assets in https://github.com/jstat/jstat.github.com I'll add you as a maintainer to that as well.

**Comment by jamescgibson** [comment]
I think I have everything set up, but when I try to push the new commit up to the 1.1 branch I get

```
ERROR: Permission to jstat/jstat.git denied to jamescgibson.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Not sure if I'm doing something wrong or what - I'm seeing a `Contributor` tag on the Github website, but no dice from the CLI.

**Comment by trevnorris** [comment]
@jamescgibson What's the output of `git remote -v`?

**Comment by jamescgibson** [comment]
```
jamescgibson	https://github.com/jamescgibson/jstat.git (fetch)
jamescgibson	https://github.com/jamescgibson/jstat.git (push)
origin	git@github.com:jstat/jstat.git (fetch)
origin	git@github.com:jstat/jstat.git (push)
```

I can `fetch` just fine, for what it's worth.

**Comment by trevnorris** [comment]
@jamescgibson sorry about that. Forgot to flip the read access to read/write access in the settings. My bad. Try again.

**Comment by jamescgibson** [comment]
No worries! I think I got it. Closed by 376eb4aed9afe3a3510df00f7183ec85ecd5ef6e

**Comment by trevnorris** [comment]
Awesome. Congrats on landing your first patch. :-)

**Pull request closed without merging**